apps/search/style/search.css

```diff
@@ -297,6 +299,14 @@ html[dir="rtl"] #suggestions-provider {
   padding: 0 1rem 0 2rem;
 }
 
+html[dir="rtl"] #suggestions li {
+  background-position: right 0.8rem center;
+}
+
+html[dir="rtl"] .local .result:after {
+  right: 0.5rem;
```

Comment:
This rule avoided the regression with the horizontal scrolling, but you are right, setting both is the best option. Thanks!